raster/r.neighbors/testsuite/test_r_neighbors.py

```diff
@@ -0,0 +1,587 @@
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+from grass.gunittest.gmodules import SimpleModule
+
+
+"""
+r.neighbors [-ac] input=name [selection=name] output=name[,name,...]
+[method=string[,string,...]] [size=integer] [title=phrase] [weight=name]
+[gauss=float] [quantile=float[,float,...]] [--overwrite] [--help]
+[--verbose] [--quiet] [--ui]
+"""
+
+
+class TestNeighbors(TestCase):
+
```

Comment:
State here or somewhere else which dataset you used. I assume it is the full one, not the basic one.